File: src/main/java/com/assets/board/model/tax/LinkedDocs.java
package com.assets.board.model.tax;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;
import jakarta.xml.bind.annotation.XmlElement;
import lombok.Data;

import java.util.List;

@Data
@XmlAccessorType(XmlAccessType.FIELD)
public class LinkedDocs {

    @XmlElement(name = "DOC")
    private List<Doc> docs;
}

File: src/main/java/com/assets/board/model/tax/LinkedDocs.kt
package com.assets.board.model.tax

import jakarta.xml.bind.annotation.XmlAccessType
import jakarta.xml.bind.annotation.XmlAccessorType
import jakarta.xml.bind.annotation.XmlElement
import lombok.Data

@Data
@XmlAccessorType(XmlAccessType.FIELD)
class LinkedDocs {
    @XmlElement(name = "DOC")
    var docs: MutableList<Doc>? = null
}
